src/api.rs

```diff
@@ -176,7 +176,7 @@ impl Context {
   pub fn frame_properties(&mut self, idx: u64) -> bool {
     let key_frame_interval: u64 = 30;
 
-    let reorder = true;
+    let reorder = false;
```

Comment:
To enable compound mode, setting "reorder = true;" is the only thing to do, correct?

Reply:
yes. that will do